### PR TITLE
chore(examples): update scroll-behavior example

### PR DESCRIPTION
**Pull request by TheAlexLichter**

### CODE REVIEW

File: examples/scroll-behavior/layouts/default2.vue [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<template>
```
Comment (pi0) [inline examples/scroll-behavior/layouts/default2.vue]: @manniL It is intentional to use `default2` as the layout name?

File: examples/scroll-behavior/pages/another-long.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <div class="container">
+    <h1>Another long page</h1>
+    <div class="spacer" style="width: 100%; height: 3000px; background: darkgreen" />
+    <nuxt-link to="/">Home page</nuxt-link>
+  </div>
+</template>
+
+<script>
+export default {
+  layout: 'default2',
```
Comment (pi0) [inline examples/scroll-behavior/pages/another-long.vue]: It seems yes 💃

File: examples/scroll-behavior/layouts/default2.vue (hunk 1 above)
Comment (manniL) [reply examples/scroll-behavior/layouts/default2.vue]: @pi0 it is :relaxed:

Comment (manniL) [reply examples/scroll-behavior/layouts/default2.vue]: related to https://github.com/nuxt/nuxt.js/issues/2738#issuecomment-379489775

**Review by pi0 — APPROVED** [review]

**Pull request merged**